src/style.css

```diff
@@ -0,0 +1,139 @@
+* {
+
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.form {
+
+  position: relative;
+  margin-top: 20px;
+  margin-right: auto;
+  margin-bottom: 0;
+  margin-left: auto;
```

Comment:
these 3 are redundant.
Review your code to optimize it and use only styles that actually matter.